spec/rubocop/cop/rspec/redundant_context_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::RedundantContext do
+  it 'registers an offense when single example inside context' do
+    expect_offense(<<~RUBY)
+      context 'when condition' do
+      ^^^^^^^^^^^^^^^^^^^^^^^^^^^ Redundant context with single example.
+        it 'does something' do
+        end
+      end
+    RUBY
+  end
+
+  it 'does not register offense when multiple examples inside context' do
```

Comment:
It is more fair to say “when the example is the only statement inside the example group. Eg another it_behaves_like or a method def would cause the cop to ignore the group.

Are there any cases with more than one statement (but one example) that we still want to flag/correct?
If not - just ignore this note, just make the description more generic, and maybe add another example with an example and some other statement in an example group.